decode_test.go

```diff
@@ -1725,13 +1725,12 @@ a:
 f:
   g:
     h:
-      i:
-        - 1
-        - 2
+      i: [1,2]
```

Comment:
Was there a special reason for this change (since both styles should actually work or maybe no)?

Reply:
It's just to test how it behaves, we use the other style down for field `x`.